utils/imagenet-process.py

```diff
@@ -37,34 +37,36 @@
 
 # create the output dir if necessary
 try:
-    os.mkdir(args.output)
-except Exception:
-    pass
+    os.makedirs(args.output, exist_ok=True)
+except Exception as e:
+    raise print(e)
 
 for ifn in glob.glob(args.input):
     name, ext = os.path.splitext(ifn)
-    name = name.split("/")[-1]
-    outputname = args.output + "/" + name + ".png"
+    name = os.path.basename(name)
+    outputname = os.path.join(args.output, name + ".png")
     print("processing", name, "as", outputname)
 
     im = PIL.Image.open(ifn)
-    im.convert("RGB")
+    im = im.convert("RGB")
     resize = torchvision.transforms.Compose(
         [torchvision.transforms.Resize(256), torchvision.transforms.CenterCrop(224)]
     )
     processed_im = resize(im)
 
     if args.normalize:
-        normalize = torchvision.transforms.Compose(
+        trans = torchvision.transforms.Compose(
```

Comment:
Nit: maybe better to call it `transform_fn` or something similar.

Reply:
Thank you for the suggestion!
d40880966c771bf1ed5ff3962c72dc6245d64987